specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/AppServiceEnvironments.json

```diff
@@ -1209,6 +1259,56 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/hostingEnvironments/{name}/outboundnetworkdependenciesendpoints": {
```

Comment:
While not functionally impactful given case insensitivity in URIs, the resourceTypeNames should follow lowerCamelCase convention

Reply:
Fixed these. Thanks!